core/src/main/java/hudson/util/TimeUnit2.java

```diff
@@ -63,7 +67,12 @@
  *
  * @since 1.5
  * @author Doug Lea
+ * @deprecated use {@link TimeUnit}. (Java 5 did not have all the units required, so TimeUnit2 was introduced because it
+ * had better conversion until Java 6 went out.)
  */
+@Deprecated
+@RestrictedSince("TODO")
+@Restricted(DoNotUse.class)
```

Comment:
Do we actually need this to be `@Restricted`? Why? `@Deprecated` accomplishes everything we need, I think.